### ISSUE TITLE
Fatal error: Uncaught TypeError: Argument 2 passed to FFraenz\PrivateComposerInstaller\Plugin::fulfillVersionPlaceholder()

### ISSUE THREAD

**Issue by DavidSteinbauer** [ffraenz/private-composer-installer#35]
We use Composer 1.10 and want to install ACF.

Here is an Excerpt of my composer file

```
    {
      "type": "package",
      "package": {
        "name": "advanced-custom-fields/advanced-custom-fields-pro",
        "version": "5.9.1",
        "type": "wordpress-plugin",
        "dist": {
          "type": "zip",
          "url": "https://connect.advancedcustomfields.com/index.php?a=download&p=pro&k=XXX&t=5.9.1"
        },
        "require": {
          "composer/installers": "^1.4",
          "ffraenz/private-composer-installer": "^5.0"
        }
      }
    },
```

```
  "require": {
    "php": ">=7.1",
    "composer/installers": "^1.8",
    "vlucas/phpdotenv": "^4.1.8",
    "oscarotero/env": "^2.1",
    "roots/bedrock-autoloader": "^1.0",
    "roots/wordpress": "5.5.1",
    "roots/wp-config": "1.0.0",
    "roots/wp-password-bcrypt": "1.0.0",
    "advanced-custom-fields/advanced-custom-fields-pro": "*",
  }

```

The Error:
```php
Fatal error: Uncaught TypeError: Argument 2 passed to FFraenz\PrivateComposerInstaller\Plugin::fulfillVersionPlaceholder() must be of the type string or null, int given, called in /Users/ds/Sites/domain.com/vendor/ffraenz/private-composer-installer/src/PrivateComposerInstaller/Plugin.php on line 182 and defined in /Users/ds/Sites/domain.com/vendor/ffraenz/private-composer-installer/src/PrivateComposerInstaller/Plugin.php:239
Stack trace:
#0 /Users/ds/Sites/domain.com/vendor/ffraenz/private-composer-installer/src/PrivateComposerInstaller/Plugin.php(182): FFraenz\PrivateComposerInstaller\Plugin->fulfillVersionPlaceholder('https://downloa...', 20200811)
#1 [internal function]: FFraenz\PrivateComposerInstaller\Plugin->handlePreInstallUpdateEvent(Object(Composer\Installer\PackageEvent))
#2 phar:///usr/local/bin/composer.phar/src/Composer/EventDispatcher/EventDispatcher.php(164): call_user_func(Array, Object(Composer\Installer\PackageEvent))
#3 phar:///usr/local/bin/composer.phar/src/Co in /Users/ds/Sites/domain.com/vendor/ffraenz/private-composer-installer/src/PrivateComposerInstaller/Plugin.php on line 239

```

**Comment by ffraenz** [comment]
Hi @DavidSteinbauer, there seems to be an issue with all numeric versions although I can't find them in your example. I released a fix in [v5.0.1](https://github.com/ffraenz/private-composer-installer/releases/tag/v5.0.1). You should no longer get the error message you mentioned. Please let me know if this works for you.

**Comment by DavidSteinbauer** [comment]
@ffraenz Thanks! After re-installing the dependencies it worked! Closing the issue!
